content/en/docs/reference/access-authn-authz/mutating-admission-policy.md

```diff
@@ -0,0 +1,215 @@
+`spec.matchConditions` of a policy will trigger the set of mutations defined.
+
+In the example above, creating a Pod will add the `mesh-proxy` initContainer mutation:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: myapp
+  namespace: default
+spec:
+  ...
+  initContainers:
+  - name: mesh-proxy
+    image: mesh/proxy:v1.0.0
+    args: ["proxy", "sidecar"]
+    restartPolicy: Always
+  - name: myapp-initializer
+    image: example/initializer:v1.0.0
+  ...
+```
+
+#### Parameter resources
+
+Parameter resources allow a policy configuration to be separate from its
+definition. A policy can define `paramKind`, which outlines GVK of the parameter
+resource, and then a policy binding ties a policy by name (via `policyName`) to a
+particular parameter resource via `paramRef`.
+
+Please refer to [parameter resources](/docs/reference/access-authn-authz/validating-admission-policy/#parameter-resources) for more information.
+
+#### `ApplyConfiguration` {#patch-type-apply-configuration}
+
+MutatingAdmissionPolicy expressions are always CEL. Each apply configuration
+`expression` must evaluate to a CEL object (declared using `Object()`
+initialization).
+
+Apply configurations may not modify atomic structs, maps or arrays due to the risk of accidental deletion of
+values not included in the apply configuration.
+
+CEL expressions have access to the object types needed to create apply configurations:
+
+- `Object` - CEL type of the resource object.
+- `Object.<fieldName>` - CEL type of object field (such as `Object.spec`)
+- `Object.<fieldName1>.<fieldName2>...<fieldNameN>` - CEL type of nested field (such as `Object.spec.containers`)
+
+CEL expressions have access to the contents of the API request, organized into CEL variables as well as some other useful variables:
+
+- `object` - The object from the incoming request. The value is null for DELETE requests.
+- `oldObject` - The existing object. The value is null for CREATE requests.
+- `request` - Attributes of the API request.
+- `params` - Parameter resource referred to by the policy binding being evaluated. Only populated if the policy has a ParamKind.
+- `namespaceObject` - The namespace object that the incoming object belongs to. The value is null for cluster-scoped resources.
+- `variables` - Map of composited variables, from its name to its lazily evaluated value.
+  For example, a variable named `foo` can be accessed as `variables.foo`.
+- `authorizer` - A CEL Authorizer. May be used to perform authorization checks for the principal (user or service account) of the request.
+  See https://pkg.go.dev/k8s.io/apiserver/pkg/cel/library#Authz
+- `authorizer.requestResource` - A CEL ResourceCheck constructed from the `authorizer` and configured with the
+  request resource.
+
+The `apiVersion`, `kind`, `metadata.name` and `metadata.generateName` are always accessible from the root of the  
+object. No other metadata properties are accessible.  
```

Comment:
Is it able to check the labels of an object?
I don't mean I want to change it.

Reply:
Next line: "This means that you cannot emit an apply configuration to make any change to an object's labels or annotations."

Reply:
yes, thanks for catching!

Reply:
OK, also: annotations?